bundles/org.openhab.binding.helios/src/main/java/org/openhab/binding/helios/internal/HeliosHandlerFactory.java

```diff
@@ -37,8 +40,15 @@
 @Component(service = ThingHandlerFactory.class, configurationPid = "binding.helios")
 public class HeliosHandlerFactory extends BaseThingHandlerFactory {
 
-    private static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Collections.unmodifiableSet(
-            Stream.of(HELIOS_VARIO_IP_2_7_TYPE, HELIOS_VARIO_IP_2_21_TYPE).collect(Collectors.toSet()));
+    private static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Collections
+            .unmodifiableSet(Stream.of(HELIOS_VARIO_IP_2_21_TYPE).collect(Collectors.toSet()));
```

Comment:
```suggestion
    private static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Collections
            .singleton(HELIOS_VARIO_IP_2_21_TYPE);
```

Reply:
AFAIK no one uses 2.7. 

New devices sold by Helios obviously run the latest version, and the 2.7 is SOAP based, which nobody wants to maintain either. The dependencies for SOAP are cumbersome, do not evolve, and render the binding complex. 2.7 must be like 7 years old

Reply:
Btw, I kept unmodifiableSet(Stream.of.... in there on purpose because it is time to support V2.30

Reply:
You could also use `Set.of()`.